app.js

```diff
@@ -7,19 +7,87 @@ const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const app = express();
 const punkAPI = new PunkAPIWrapper();
 
+const axios = require('axios').default;
+
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
-// Register the location for handlebars partials here:
-
-// ...
-
-// Add the route handlers here:
+fetch('https://dummyjson.com/auth/login', {
+  method: 'POST',
+  headers: { 'Content-Type': 'application/json' },
+  body: JSON.stringify({
+    username: 'kminchelle',
+    password: '0lelplR'
+  })
+})
+  .then(res => res.json() && console.log('auth worked'))
+  .catch(error => console.log('error') && console.log('auth error'));
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  const itemList = axios.get('https://dummyjson.com/products');
+  let productData;
+  let itemNames = [];
+  itemList
+    .then(item => {
+      const { data, ...rest } = item;
+      productData = data.products;
+      return productData;
+    })
+    .then(() => {
+      for (let i = 0; i < 25; i++) {
```

Comment:
1. avoid for loop with let i =0 sync, here you have define the length everytime. (use better available options .forEach, for(const each of array)